.flake8

```diff
@@ -45,7 +45,7 @@ ignore =
 # ========================
 per-file-ignores =
     Bio/*:F401,F841,D105,B009,B010,B011,C812,C815
-    Tests/*:F401,F841,D101,D102,D103,W504,B009,B010,B011,C812,BLK100
+    Tests/*:F401,F841,D101,D102,D103,W504,B009,B010,B011,C812
```

Comment:
Can you also change the entry five lines down to match please.

Reply:
Sorry, missed it. Done.